README.adoc

```diff
@@ -29,7 +29,7 @@ You can take a look at the link:sonar-project.properties[sonar-project.propertie
 - https://docs.sonarqube.org/latest/analysis/github-integration/[GitHub Integration in SonarQube]
 - https://docs.sonarqube.org/latest/analyzing-source-code/languages/c-family/#analysis-cache[Configuring C, C++ and Objective-C analysis cache]
 - https://docs.sonarqube.org/latest/analyzing-source-code/languages/c-family/#parallel-code-scan[Configuring multithreaded execution]
```

Comment:
Didn't we also agree to remove the mult-threading and cache links? Am I missing something?

See the discussion in sonarsource-cfamily-examples/linux-autotools-gitlab-ci-sc#7